pinc/Project.inc

```diff
@@ -979,6 +981,31 @@ class Project
             || that_user_is_proj_facilitator($username));
     }
 
+    public function user_can_delete_nonpage_images($username)
+    {
+        if (is_null($username)) {
+            return false;
+        }
+        // SAs can always delete nonpage images
+        if (that_user_is_a_sitemanager($username)) {
+            return true;
+        }
+
+        // PMs/PFs can only delete nonpage images in New
+        // or P1 Unavailable stages
+        if ($this->can_be_managed_by_user($username)) {
+            if ($this->state == PROJ_NEW || $this->state == PROJ_P1_UNAVAILABLE) {
+                return true;
```

Comment:
Up to you if you think this is more clear than just
```
return $this->state == PROJ_NEW || $this->state == PROJ_P1_UNAVAILABLE;
```

Reply:
I waffled over this one a bit, but ultimately came down on the side of making the change. But ... tomorrow.

Reply:
Done and done! Sandbox has been updated (after each change, I like to verify that I didn't break anything making the changes).